xml/operations-maintenance-update_maintenance.xml

```diff
@@ -178,6 +178,20 @@ ardana-update.yml --limit <replaceable>TARGET_NODE_NAME</replaceable></screen>
        Reboot the node as described in <xref linkend="rebootNodes"/>.
       </para>
      </step>
+     <step>
+      <para>
+       Please run <filename>spark-start.yml</filename> playbooks after the node comes up after a reboot:
```

Comment:
* Do not use "please"
* Should "playbooks" be in plural? There is just one file given here.
* Suggestion to fix the sentence order and remove the second "after": `When the node comes up after a reboot, run the playbook[s?] <filename>spark-start.yml</filename>:`
* Also fix the line length.